crates/re_log_types/src/time.rs

```diff
@@ -7,12 +7,24 @@ use time::OffsetDateTime;
 pub struct Time(i64);
 
 impl Time {
+    // NOTE: Even though `instant` should work on wasm, `elapsed` is broken.
+    // See: https://github.com/sebcrozet/instant/issues/49
+    // We would like to switch to `web-time`, but that's broken in a different way
+    // See issues in: https://github.com/rerun-io/rerun/pull/2093
+    //
+    // So we implement what elapsed is supposed to do ourselves.
```

Comment:
nit: this is a comment about the implementation and so should be within the function, not outside it

Reply:
Ahh; good call. I had started with two separate implementations so this made more sense, but will move into the function.